Corgibytes.Freshli.Cli/Functionality/Git/CachedGitSourceRepository.cs

```diff
@@ -6,6 +6,7 @@
 using Corgibytes.Freshli.Cli.DataModel;
 using Corgibytes.Freshli.Cli.Exceptions;
 using Corgibytes.Freshli.Cli.Resources;
+using Newtonsoft.Json;
```

Comment:
In an earlier PR we wanted to move away from Newtonsoft.Json. Do we still want to?
I did a bit of googling for how to achieve the same thing in System.Text.Json and according to this stackoverflow you need JsonInclude for private properties:
```csharp
[System.Text.Json.Serialization.JsonInclude]
[System.Text.Json.Serialization.JsonPropertyName("animals")]
public string AnimalForSerializer
{
    get => AnimalForMe.ToString() + "s";
    private set => AnimalForMe = Enum.Parse<Animal>(value.TrimEnd('s'));
}
```

https://stackoverflow.com/questions/58475629/what-is-the-equivalent-of-newtonsoft-jsons-json-nets-jsonproperty-field-in-s

Reply:
I would love to get rid of Newtonsoft.Json, but it's used by Hangfire. I don't think the System.Text.Json attributes will work the same way.